ibtk/src/solvers/impls/CCPoissonPointRelaxationFACOperator.cpp

```diff
@@ -65,96 +65,184 @@
 
 // FORTRAN ROUTINES
 #if (NDIM == 2)
-#define GS_SMOOTH_FC IBTK_FC_FUNC(gssmooth2d, GSSMOOTH2D)
-#define RB_GS_SMOOTH_FC IBTK_FC_FUNC(rbgssmooth2d, RBGSSMOOTH2D)
-#define VC_CELL_GS_SMOOTH_FC IBTK_FC_FUNC(vccellgssmooth2d, VCCELLGSSMOOTH2D)
-#define VC_CELL_RB_GS_SMOOTH_FC IBTK_FC_FUNC(vccellrbgssmooth2d, VCCELLRBGSSMOOTH2D)
+#define CONST_DC_GS_SMOOTH_FC IBTK_FC_FUNC(const_dc_gssmooth2d, CONST_DC_GSSMOOTH2D)
```

Comment:
Let's stick with the original names for the original functions. Although unlikely, it is possible that they are used in other places.

Reply:
Since @amneetb and @drwells also like the new names, I will mark this as resolved.